Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalHEMuonFilter_cff.py

```diff
@@ -0,0 +1,15 @@
+import FWCore.ParameterSet.Config as cms
+
+#-------------------------------------------------
+#AlCaReco filtering for HCAL HEMuon:
+#-------------------------------------------------
+import HLTrigger.HLTfilters.hltHighLevel_cfi
+ALCARECOHcalCalHEMuonFilterHLT = HLTrigger.HLTfilters.hltHighLevel_cfi.hltHighLevel.clone(
+    HLTPaths = cms.vstring('SingleMuon'),
```

Comment:
Hello @bsunanda , this is HLT paths will not work in real production;
it will look for HLT path name **identical** to SingleMuon, but there is no such path.

Reply:
@tocheng Did you change the global tag so that when I replace HLTPaths to eventSetupPathsKey

Reply:
@bsunanda 
I thought in the last e-mail exchange, you mentioned you can use HcalCalHBHEMuonFIlter as the eventSetupPathsKey for this new AlCaReco.
To add new eventSetupPathsKey for new AlCaReco, we need to make new tag for AlCaReco triggerbit and new offline GTs and make PR to change offline GTs. Since you have another HcalCalLowPUHBHEMuonFilter, it is easier to add these two in one go.